packages/terra-framework-docs/src/terra-dev-site/doc/data-grid/FlowsheetDataGrid.4/Examples.6/ColumnHighlight.6.doc.mdx

```diff
@@ -0,0 +1,9 @@
+import ColumnHighlight from "./ColumnHighlight";
+
+# Flowsheet Data Grid with column highlighting
+
+### Description
+
+This example demonstrates a basic [Flowsheet Data Grid](/components/cerner-terra-framework-docs/data-grid/flowsheet-data-grid/about) with column highlighting.
```

Comment:
These examples should explain more than what feature they show.

Especially with niche features like this, there should be information accompanying the examples about when these features should be used and any other notes consumers should be aware of when using the feature. It's fine if this information repeats things already included on the About page.

Reply:
Updated description regarding the properties to be used during highlighting: https://github.com/cerner/terra-framework/pull/1985/commits/a421f62563953de2c0342e27929a00b95e7c080b